### ISSUE TITLE
01:42 You see a vial. It is empty. It weighs 1.80 oz. // can't be refilled with blood, poison etc

### ISSUE THREAD

**Issue by Felipe1993** [TwistedScorpio/OTHire#208]
In real tibia at 772 version when you got  a vial. It is empty.
It weighs 1.80 oz.  if you put a monster above blood or poison the vial can be refilled with blood poison pis etc and this is not working at OTHIRE 1.0

regards

**Comment by peonso** [comment]
Several corpses missing tag `<attribute key="fluidSource" value="blood"/>`.

**Comment by tornadia** [comment]
Also, this isn't really relevant, but the data folder is incomplete. The shovel system might not function properly, given that it is using a moderately recent function, and the ID it is meant to work with is beyond the itemids scope.

**Comment by Felipe1993** [comment]
i should add this <attribute key="fluidSource" value="blood"/>. to coprse files at items.xml? it's all ?
if you tell me that it's all the problem i can do it a pull a request

**Comment by tornadia** [comment]
One question, I see you added fluidSource='blood' to slain skeletons, which are undead... By the logic ordinance of them being "undead" and "skeletons" (i.e. floating bone parts), shouldn't they have *no* fluid source? I really don't remember how it was back in the day anyway.

**Comment by peonso** [comment]
Yeah, skeleton shouldn't have blood, I added the items.xml made by you guys, but it still need working regarding proper fluid sources to corpses (removing those that shouldn't have a fluidSource).

For reference, leaked file items: https://pastebin.com/GNVaTB9k - https://pastebin.com/wZCcuJgg

**Comment by Felipe1993** [comment]
thanks mate c: soo i will try to make it again today i was very tired when i edited the xml
